test/base/CustomExportsBaseSpec.scala

```diff
@@ -63,6 +66,12 @@ trait CustomExportsBaseSpec
   val mockNrsService: NRSService = mock[NRSService]
   val mockItemsCachingService: ItemsCachingService = mock[ItemsCachingService]
 
+  implicit val hc: HeaderCarrier =
```

Comment:
I'm not sure that it's the correct place for HeaderCarrier as implicit. Too big scope. I prefer like before, in specific test if you need it.

Reply:
I actually side with @raghu1978 on this one. 
Happy either way.

Reply:
Just my thoughts, sometimes you don't want to have authorisation inside. We can stay with this